src/services/device-management/application.service.ts

```diff
@@ -121,14 +123,11 @@ export class ApplicationService {
         let orderBy = `application.id`;
         if (
             query.orderOn != null &&
-            (query.orderOn == "id" ||
-                query.orderOn == "name" ||
-                query.orderOn == "updatedAt")
+            (query.orderOn == "id" || query.orderOn == "name" || query.orderOn == "updatedAt")
```

Comment:
Can we make these === instead of == ? Even though I know it's not your code..

Reply:
Done (left != null checks as this also checks for undefined